lascli/actions/workflows.py

```diff
@@ -0,0 +1,262 @@
+                'medium': 0.5,
+                'low': 0.3,
+            }
+        }
+
+    return {field: transform(field) for field in field_config}
+    
+
+@wrap_output(start_msg='Creating secrets ...', end_msg='Done')
+@capture_return(dest=created_ids['secrets'])
+def create_secrets(las_client: Client, create_tag: str, username: str = None, password: str = None):
+    docker_secret = {}
+    if username and password:
+        docker_secret = las_client.create_secret(
+            data={
+                'username': username,
+                'password': password 
+            },
+            name='Docker credentials',
+            description=create_tag,
+        )['secretId']
+        
+    cradl_secret = las_client.create_secret(
+        data={
+            'LAS_CLIENT_ID': las_client.credentials.client_id,
+            'LAS_CLIENT_SECRET': las_client.credentials.client_secret,
+            'LAS_AUTH_ENDPOINT': las_client.credentials.auth_endpoint,
+            'LAS_API_ENDPOINT': las_client.credentials.api_endpoint
+        },
+        name='Cradl credentials',
+        description=create_tag,
+    )
+        
+    return docker_secret.get('secretId'), cradl_secret['secretId']
+
+
+@wrap_output(start_msg='Creating remote component and field config asset ...', end_msg='Done')
+@capture_return(dest=created_ids['assets'])
+def create_assets(las_client: Client, model_id: str, create_tag: str):
+    component = las_client.create_asset(
+        content=requests.get(COMPONENT_ASSEST_URI).content,
+        name='Manual validation component',
+        description=create_tag,
+    )
+    
+    field_config = las_client.create_asset(
+        content=json.dumps(create_field_config(las_client, model_id)).encode('utf-8'),
+        name='Field configuration',
+        description=create_tag,
+    )
+
+    return component['assetId'], field_config['assetId']
+
+
+@wrap_output(start_msg='Creating dataset ...', end_msg='Done')
+@capture_return(dest=created_ids['datasets'])
+def create_dataset(las_client: Client, name: str, create_tag: str):
+    dataset = las_client.create_dataset(name=name, description=f'Dataset for {name} model. {create_tag}')
+    return dataset['datasetId']
+    
+
+@wrap_output(start_msg='Creating transitions ...', end_msg='Done')
+@capture_return(dest=created_ids['transitions'])
+def create_transitions(
+    las_client: Client, 
+    cradl_secret_id: str, 
+    parameters: dict,
+    create_tag: str, 
+    created_ids: dict, 
+    docker_secret_id: str = None,
+):
+    docker_auth_details = {'secretId': parameters['secret_id']} if parameters.get('secret_id') else {}
+
+    common_params = {
+        'environmentSecrets': [cradl_secret_id],
+        'environment': {
+            'MODEL_ID': parameters['model_id'],
+            'FIELD_CONFIG_ASSET_ID': parameters['field_config_asset_id'],
+            'DATASET_ID': parameters['dataset_id'],
+        },
+        **docker_auth_details
+    }
+    
+    print(common_params)
```

Comment:
Is this a debug print or a print we actually want?

Reply:
Ooops!